### PR TITLE
Consistent navigation and headlines

### PR DESCRIPTION
**Pull request by cebe**
Use the same concept for navigation and headlines throughout the site.
Added documentation about the concepts.

- [x] adjust CSS to make sure spacing is consistent among all pages.
- [x] adjust static pages headline styling

### CODE REVIEW

File: assets/src/scss/3-global/_content.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
 .content {
-  @include rem(margin-top, 50px);
+  margin-top: 0;
+  //@include rem(margin-top, 50px);
```
Comment (samdark) [inline assets/src/scss/3-global/_content.scss]: ?

File: assets/src/scss/5-pages/_guide.scss [hunk 2]
```diff
@@ -1,5 +1,6 @@
 .guide-content {
-  @include rem(margin-top, 50px);
+  margin-top: 0;
+  //@include rem(margin-top, 50px);
```
Comment (samdark) [inline assets/src/scss/5-pages/_guide.scss]: ?

File: views/extension/create.php [hunk 3]
```diff
@@ -27,6 +20,8 @@
 
         <div class="col-sm-9 col-md-10 col-lg-10" role="main">
 
+            <h1>Create an Extension</h1>
```
Comment (samdark) [inline views/extension/create.php]: Looks like title. Why it's hardcoded?

Comment (cebe) [inline views/extension/create.php]: will make a concept for title too and adjust it if needed.

File: assets/src/scss/5-pages/_guide.scss (hunk 2 above)
Comment (cebe) [inline assets/src/scss/5-pages/_guide.scss]: will clean up comments when done. Commented because not sure if I need to revert it.

**Comment by samdark** [comment]
Applied headers where it was appropriate. Implemented version selector for wiki and extensions.

**Pull request merged**